changelog.d/8502.misc

```diff
@@ -0,0 +1 @@
+Increase default upload size limit from 10M to 50M.
```

Comment:
given that this is a user-visible change, it should probably be a `.feature` file

Reply:
Thank you for the review!
I wasn't entirely sure about the .misc, changed it to .feature as suggested. :D